src/components/settings/SettingsAccessTab/SettingsAccessLinkManagement.jsx

```diff
@@ -83,11 +71,8 @@ const SettingsAccessLinkManagement = ({ enterpriseUUID }) => {
           itemCount={links.length}
           // eslint-disable-next-line no-unused-vars
           tableActions={(i) => (
-            // TODO: Consider making this its own component with logic
-            <StatefulButton
-              {...generateLinkButtonProps}
-              onClick={handleGenerateLinkSuccess}
-              disabledStates={!isLinkManagementEnabled ? Object.keys(generateLinkButtonProps.labels) : undefined}
+            <SettingsAccessGenerateLinkButton
+              onSuccess={handleGenerateLinkSuccess}
```

Comment:
Currently, when the "Access via Link" section is disabled via the `Form.Switch`, it disables the "Generate link" button as it doesn't make sense to generate a link when "Access via Link" is turned off.

Are we no longer disabling the button here when link management isn't enabled?

![image](https://user-images.githubusercontent.com/2828721/148249585-c66a6e76-68a8-4f66-a6d0-76379aec36d7.png)

Reply:
Complete oversight on my part! Good catch!